server/models.py

```diff
@@ -33,12 +33,12 @@ class Position(EmbeddedDocument):
 
     TODO: Make this more modular, and independent of max_size
     """
-    x = IntField(required=True, choices=range(1, max_size + 1))
-    y = IntField(required=True, choices=range(1, max_size + 1))
-    z = IntField(required=True, choices=range(1, max_size + 1))
+    x = IntField(required=True, choices=range(0, max_size))
```

Comment:
just to confirm that this is what you intended because python ranges are [inclusive, exclusive), you want to allow points to be 0, and have them strictly be at most `max_size - 1`?

Reply:
Yup map indices go from (0, 0, 0) to (47, 47, 47) where max size is 48.

Reply:
ok then there are just some conflicts to resolve quickly and then i will approve

Reply:
Fixed